elements/with-custom-param/index.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Custom element with a custom param type for WPBakery.
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+// Include the custom param type registration and styles.
+require_once __DIR__ . '/custom-param.php';
+
+/**
+ * Registers new element with WPBakery.
+ *
+ * Here we specify all the parameters that help us see and edit the element in WPBakery editors.
+ *
+ * @see https://kb.wpbakery.com/docs/inner-api/vc_map/ for all possible vc_map parameters.
+ */
+add_action('vc_before_init', function () {
+	vc_map([
+		'name'        => __( 'With Custom Param', 'vc-dev-example' ),
```

Comment:
I think we don't want to mention vc anymore, and as we don't have any bc worries here, please make this wpb-dev-example not vc-dev-example